File: JAVA/PARCIAL II/EVA07-POO-TrujilloDavid/src/paquete1/Persona.java

package paquete1;


public abstract class Persona implements Imprimir{
    protected int id;
    protected String nombre;
    protected int edad;

    public Persona(int id, String nombre, int edad) {
        this.id = id;
        this.nombre = nombre;
        this.edad = edad;
    }
    
    public abstract String Concentrarse();
    public abstract String Viajar();
    
    @Override
    public String toString(){
        return "Id: "+id+" Nombre: "+nombre+" Edad: "+edad;
    }

}

File: JAVA/PARCIAL II/EVA-POO-TrujilloDavid/EVA-POO-TrujilloDavid/Persona.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EVA_POO_TrujilloDavid
{
	internal abstract class Persona
	{
		private int id;
		private string nombre;
		private int edad;

		public Persona( int id, string nombre, int edad)
		{
			this.id = id;
			this.nombre = nombre;
			this.edad = edad;
		}

		public int Id
		{
			get { return id; }
			set { id = value; }
		}
		public string Nombre
		{
			get { return nombre; }
			set { nombre = value; }
		}
		public int Edad
		{
			get { return edad; }
			set { edad = value; }
		}
		public abstract string Concentrarse();
		public abstract string Viajar();

		public override string ToString()
		{
			return "Id: " + id + " Nombre: " + nombre + " Edad: " + edad;
		}
	}
}
